example-app/build.gradle

```diff
@@ -19,6 +19,7 @@ android {
         vectorDrawables {
             useSupportLibrary true
         }
+        buildConfigField("String", "KRONOR_MERCHANT_TOKEN", project.property("KRONOR_MERCHANT_TOKEN"))
```

Comment:
I think the `local.properties` file is better suited. The file is not version controlled too.

Reply:
I am guessing you meant that I should put the key in `local.properties` file. I have read since that file is managed by android studio, to not manipulate it ourselves